eventstore/store.go

```diff
@@ -84,6 +242,25 @@ func (s *Store) readTxn(m *Model, f func(txn *Txn) error) error {
 	return nil
 }
 
+// Close closes the store
+func (s *Store) Close() {
+	s.cancel()
+	s.localEventsBus.bus.Discard()
+	s.stateChanged.bus.Discard()
+	s.datastore.Close()
+	if s.ownThreadService {
```

Comment:
Only close the Threadservice if we created a default one. If a custom one was provided, let the original owner deal with it since can being used for other things. Opinions?

Reply:
It might be cleaner to have a _make default thread service_ method that is available to the user. Then they just use that instead of the option. In either case, they have to close it down.

Reply:
Feels better. Having the Store provide a method to build a Threadservice is quite heavy and seems out of scope. Created #87.